queries.sql

```diff
@@ -8,3 +8,36 @@ SELECT name, escape_attempts FROM animals WHERE animals.weight_kg > 10.5;
 SELECT * FROM animals WHERE animals.neutered = true;
 SELECT * FROM animals WHERE animals.name != 'Gabumon';
 SELECT * FROM animals WHERE animals.weight_kg >= 10.4 AND animals.weight_kg <= 17.3;
+
+START TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+START TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+START TRANSACTION;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+START TRANSACTION;
+DELETE FROM animals WHERE animals.date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(escape_attempts) FROM animals WHERE animals.escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, SUM(escape_attempts) as total_escape_attempts FROM animals GROUP BY neutered;
+SELECT MIN(weight_kg), MAX(weight_kg) FROM animals;
+SELECT date_of_birth, AVG(escape_attempts) FROM animals GROUP BY date_of_birth HAVING date_of_birth BETWEEN '1990-01-01' AND '2000-01-01';
```

Comment:
- [ ] 👍 Your SQL query is functional and retrieves the average number of escape attempts for animals born between 1990 and 2000, grouped by their respective dates of birth. While this query is useful for analyzing escape behavior within that date range, it doesn't fully meet the project requirement, which asks for the average number of escape attempts per animal type born between 1990 and 2000. To align your code with the project requirements, please consider updating your query to calculate the average escape attempts per animal type within the specified date range. Great work so far! 👏📊
- [ ] Please also update the screenshot in the PR description if necessary!